pkg/image/api/v1/types.go

```diff
@@ -34,6 +34,12 @@ type Image struct {
 	DockerImageLayers []ImageLayer `json:"dockerImageLayers"`
 	// Signatures holds all signatures of the image.
 	Signatures []ImageSignature `json:"signatures,omitempty"`
+	// DockerImageSignatures provides the signatures as opaque blobs.
+	DockerImageSignatures [][]byte `json:"dockerImageSignatures,omitempty"`
+	// DockerImageManifestMediaType specifies the mediaType of manifest.
```

Comment:
Please mention that this is an attribute of manifest schema v2. The same for `DockerImageConfig`.